.travis.yml

```diff
@@ -1,15 +1,26 @@
+
 language: node_js
 node_js: "stable"
 
+python:  3.6
+cache: pip
+
 before_install:
   - npm install -g npm@latest
+  - pip install --upgrade pip
+  - pip install -U pip wheel
```

Comment:
these 2 lines are failing for some reason, there's no real urgent need to do this though so you  can remove them